docs/supplements/ArtifactTemplate.md

```diff
@@ -0,0 +1,159 @@
+# Artifact README Template
+
+> Artifact for paper: **<Paper Title>**
+
+## Preprint
+
+<!-- 
+For single-blind review (reviewers know author identities):
+  Provide a direct link to the online preprint 
+-->
+[Download the preprint from ArXiv/DOI/etc.](https://arxiv.org/abs/xxxx.xxxxx)
+
+<!-- 
+For double-blind review (reviewers don't know author identities):
+  Include the PDF directly in the artifact and reference it locally
+  Do NOT include any identifying information or links that would reveal author identities
+-->
+[Access the anonymized preprint](./docs/preprint.pdf)
+
+<!-- Choose ONE of the above options based on your venue's review model and DELETE the other -->
+
+## Table of Contents
+
+1. [Overview](#overview)
+2. [Claim-to-Artifact Traceability](#claim-to-artifact-traceability)
+3. [Repository Structure](#repository-structure)
+4. [Getting Started](#getting-started)
+5. [Usage](#usage)
+6. [Artifact Contents](#artifact-contents)
+7. [Citations](#citations)
+8. [License](#license)
+9. [Contact](#contact)
+
+## Overview
+
+Briefly describe the artifact, its purpose, and how it supports the claims or contributions of the paper.
+
+## Claim-to-Artifact Traceability
+
+This table maps key paper sections or claims to corresponding artifact components. It ensures reproducibility and transparency.
+
+| Paper Section / Claim                 | Artifact Component        | Location                   |
+| ------------------------------------- | ------------------------- | -------------------------- |
+| Claim 1 (e.g., RQ1: data stats)       | Data preprocessing script | `scripts/preprocess.py`    |
+| Claim 2 (e.g., RQ2: model evaluation) | Evaluation notebook       | `notebooks/evaluate.ipynb` |
+| Claim 3 (e.g., RQ3: visualization)    | Plotting utility          | `scripts/plot_results.py`  |
+| ...                                   | ...                       | ...                        |
+
+Include a screenshot illustrating this mapping:
```

Comment:
This says "You should include such a screenshot" which I find confusing. Maybe you meant to say, as a comment, "Here is a screenshot illustrating this mapping".

Let's include the link to the artifact itself please, in addition to the screenshot. "Here is a screenshot illustrating this mapping, taken from [REPO](link)".